db/migrations/20190606140324_artists.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex, Promise) {
+  return knex.raw(`
+    CREATE TABLE artists (
+      id INT AUTO_INCREMENT PRIMARY KEY NOT NULL,
+      name VARCHAR(50) UNIQUE
```

Comment:
Column `name` should be marked as `NOT NULL`.
Some DB engines assume `NOT NULL` as the default value, but since you have it specified for `id` column, we need to be consistent.